### DISCUSSION TITLE
Default Page Builder - change any content on your homepage template in Breeze..

### DISCUSSION THREAD

**Discussion by sdomeradzki** [breezefront/community#13] [1. Help]
Hi,

I have problem with Breeze Evolution. When I change any content on homepage (building as page builder). I lost viewport. Can you check on your demo?
I not have any other extensions, only from swisslabs. Can you check?

In attachment show you what is wrong.
<img width="1150" alt="Zrzut ekranu 2022-09-21 o 20 19 21" src="https://user-images.githubusercontent.com/110283346/191581517-d4ee346d-6ead-489c-a6ea-99973a16de61.png">
<img width="1680" alt="Zrzut ekranu 2022-09-21 o 20 21 13" src="https://user-images.githubusercontent.com/110283346/191581553-0def4098-d24e-4275-9d88-d8dba0de5443.png">

**Comment by vovayatsyuk** [comment]
Looks like a breaking change in magento 2.4.5. Magento now wraps columns in the additional wrapper and our custom styles don't work properly. Will fix it shortly and let you know.

The other part that I can't reproduce is the blue titles and centered description on the headphones. Is it an issue too? Or these are your changes and they are correct?

**Comment by sdomeradzki** [comment]
Hi, thanks for fast reply.

Yes this is Magento 2.4.5. and "_blue titles and centered description on the headphones_" aslo is issue. This is original code from your theme. I m not change any css.

**Comment by sdomeradzki** [comment]
Will be great if you can help me, because we have already store on Magento 2.4.5. and we can't put any new content to cms pages :( We will wait for you :)

**Comment by vovayatsyuk** [comment]
Here is a fix for columns: https://github.com/breezefront/theme-frontend-breeze-blank/commit/2e44a9778f59d98bdfca656e4c5060a30324804a

Don't want to tag the release right now because I can't reproduce the issue with blue text.

Do you use the latest version of breeze? Try to re-run the installer if yes.

**Comment by sdomeradzki** [comment]
Hi, @vovayatsyuk 
I updated, thanks.
n desktop now look good, but for mobile still are problems.

Please look attachment.
<img width="452" alt="Zrzut ekranu 2022-09-22 o 14 29 47" src="https://user-images.githubusercontent.com/110283346/191747451-a180759c-11ea-485d-8b7f-25d15e4d6e1c.png">
<img width="452" alt="Zrzut ekranu 2022-09-22 o 14 29 31" src="https://user-images.githubusercontent.com/110283346/191747464-f8d8bc9e-73b7-465e-bc36-70a6f6358b4d.png">

**Comment by vovayatsyuk** [comment]
I forgot to mention another related commit I made yesterday: https://github.com/breezefront/theme-frontend-breeze-blank/commit/0a0fde80e6449d6612f64d46b81cf44cfa8bc531

**Comment by sdomeradzki** [comment]
Great! @vovayatsyuk work fine thanks!
